src/button/tests/functional/Button.ts

```diff
@@ -0,0 +1,73 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+
+function getPage(remote: any) {
+	return remote
+		.get('http://localhost:9000/_build/common/example/?module=button')
+		.setFindTimeout(5000);
+}
+
+registerSuite({
+	name: 'Button',
+
+	'button should be visible'(this: any) {
+		return getPage(this.remote)
+			.findByCssSelector('button:first-of-type')
+		// `isDisplayed` is broken in Safari 10, commented out for now. See https://github.com/SeleniumHQ/selenium/issues/3029
```

Comment:
Rather than leaving this in, can we please remove it in a separate commit? Then it can be added back in again as and when it's appropriate to do so. I think it's better not to have commented out code in our master branch.